### DISCUSSION TITLE
Google Lighthouse issue: Registers an unload listener

### DISCUSSION THREAD

**Discussion by GaurangMistry** [dnnsoftware/Dnn.Platform#5914] [General]
We have many sites in DNN and we are creating all sites perfectly with Google LightHouse. In Google LightHouse, we are facing one issue common in all sites is "Registers an unload listener". This is coming from Default Jquery file which DNN used.

I searched in google and got many suggestions to delete "Unload" from addEventListener method from Jquery or use "visibilitychange" but it's not work. We also take help of Jquery experts but not get expected result. So, we wish if you have any idea about it and you can directly work with us on project to fix this issue.
Steps to reproduce?

1.     Install Google Chrome Light House plugin
2.     Open website: https://www.shopsmart.website/ in Google Chrome
3.     Right-Click on website and select "Inspect".
4.     There is one option called "Lighthouse". Select it and click on "Analyze page load".
5.     Select "Best Practices" and you will get error for "Registers an unload listener".

I searched in google and got many suggestions to delete "Unload" from addEventListener method from Jquery or use "visibilitychange" but it's not work. We also take help of Jquery experts but not get expected result.

![2024-01-10_16-07-58](https://github.com/dnnsoftware/Dnn.Platform/assets/33444274/b8da9a96-35ee-4575-8e4a-1c7b5a302137)

![2024-01-10_16-08-40](https://github.com/dnnsoftware/Dnn.Platform/assets/33444274/8a491bec-baf3-47fe-b17c-05245c0784f9)

![2024-01-10_16-09-04](https://github.com/dnnsoftware/Dnn.Platform/assets/33444274/5bf00631-d0b5-4bac-b84f-6ad1621c9cc9)

Please suggest for this fixing

Thanks

**Comment by uzmannazari** [comment]
nobody knows how to fix this? i have the same problem on the dnn sites
